using/config.json

```diff
@@ -6,5 +6,12 @@
     "path": "using/README.md",
     "title": "Using Exercism",
     "blurb": "Learn how to make the most of our experience on Exercism"
+  },
+  {
+    "uuid": "7b753e2a-bd68-4d68-bdd8-c356de4342b6",
+    "slug": "solving-exercises/locally",
```

Comment:
I'm not sure `/` in these paths will work with how Rails will build the routes. I think it's better to use `-` so Google interprets the string as a sentence.

Reply:
It is used in other configs too: https://github.com/exercism/docs/blob/main/building/config.json#L19